scipy/interpolate/_bsplines.py

```diff
@@ -271,6 +276,22 @@ def tck(self):
         """
         return self.t, self.c, self.k
 
+    @property
```

Comment:
A comment here or above in the init would help, something like
```suggestion
    # Under the hood, self._c and self._t are always saved as numpy array
    # because they are used in a C extension expecting numpy arrays.
    @property
```